common_python/input.py

```diff
@@ -56,6 +57,10 @@ def __init__(self, *args, **kwargs):
 
         self.o_input_data = InputData()
 
+        if 'microsoft' in uname().release:
+            self.gui_mode = False
+            return
+
         tk.Tk.__init__(self, *args, **kwargs)
```

Comment:
would it help to move lines 64 until 83 into the if-statement after line 54?
line 58 needs then to be the first line of this init-function

Or even move lines 66 until 83 into the start_gui() function and leave 64 in the if-statement after line 54?

for macOS that seams to work. I wonder if it also works on WSL?

Reply:
as discussed, this is an other topic which is tackled in #73. this here is good to go! 💯